src/geo/LatLng.js

```diff
@@ -135,3 +134,302 @@ export function toLatLng(a, b, c) {
 	}
 	return new LatLng(a, b, c);
 }
+
+/**
+ * @param {{ lng: any; lat: any; alt: any; }} latlng
+ */
+export function wrapLatLng(latlng) {
+	var lng = this.wrapLng ? Util.wrapNum(latlng.lng, this.wrapLng, true) : latlng.lng,
+	lat = this.wrapLat ? Util.wrapNum(latlng.lat, this.wrapLat, true) : latlng.lat,
+	alt = latlng.alt;
+
+	return new LatLng(lat, lng, alt);
+}
+
+/*
+ * @class LatLngBounds

```

Comment:
Why merge `LatLngBounds` into `LatLng`? They should be separate.

Reply:
LatLng toBound uses toLatLngBounds, which references LatLng.  I could not discover how to escape this circular import unless I merged them. Do you have a suggestion to fix?

Reply:
Also verified documentation built, note I did have issue with build on my platform but raised in different issue. (#7271)

Reply:
API-breaking changes are frowned upon. I'd go with the `LatLngBounds` file adding functionality to `LatLng`, e.g. `LatLng.include({ toLatLngBounds: function(...){...} })`. Look at how the code for `GeoJSON` adds stuff to markers and polylines.

This approach would mean that a `LatLng` imported stand-alone would not have the `toLatLngBounds()` method, and that  users would need to import `LatLngBounds` as well. I think this is a lesser problem than always bundling both `LatLng` and `LatLngBounds` together.

Reply:
@IvanSanchez , I have separated LatLng from LatLngBounds.  As you advised, leverage the Class.Extend to isolate toBounds(...) from LatLng and avoid circular dependency.  geo/index will export toBounds extended as LatLng to hopefully avoid api change.